framework/src/errors.ts

```diff
@@ -15,6 +15,7 @@
 
 export class FrameworkError extends Error {
 	public name: string;
+	public code = 'framework';
```

Comment:
should we use error code like
framework: 1000-2000(range)
so that user could handle it better? would be nice to standardise code

Reply:
I thought of this as well but then we need to have a translation from number code to meaningful

Reply:
yea since we will need translation, i just made it to string so that without any reference, at least it has some meaning